ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -138,17 +153,22 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, int maxConn
             connId = selector.connect(new InetSocketAddress(host, port.getPort()), networkConfig.socketSendBufferBytes,
                 networkConfig.socketReceiveBufferBytes, port.getPortType());
             connectionTracker.startTrackingInitiatedConnection(host, port, connId);
+            requestMetadata.connectionCheckOutAttempts++;
```

Comment:
the comment is on `connectionCheckOutAttempts`. what is your definition of a connection checkout attempt?